4 Ord Queries Exercise - Driver Finder/driverFinder-rt/src/com/tridiumuniversity/driverFinder/BDriverFinder.java

```diff
@@ -0,0 +1,77 @@
+package com.tridiumuniversity.driverFinder;
+
+import javax.baja.collection.BITable;
+import javax.baja.collection.Column;
+import javax.baja.collection.ColumnList;
+import javax.baja.collection.TableCursor;
+import javax.baja.naming.BOrd;
+import javax.baja.nre.annotations.NiagaraAction;
+import javax.baja.nre.annotations.NiagaraType;
+import javax.baja.query.BQueryResult;
+import javax.baja.sys.*;
+import java.util.logging.Logger;
+
+@NiagaraType
+@NiagaraAction(name = "find", parameterType = "BQueryImplementation", defaultValue = "BQueryImplementation.bql")
```

Comment:
Using `BQueryImplementation` is a really neat approach. It reminds me of the Strategy design pattern.